addons/cargo/functions/fnc_paradropItem.sqf

```diff
@@ -89,6 +90,7 @@ _itemObject setVelocity ((velocity _vehicle) vectorAdd ((vectorNormalized (vecto
             private _smoke = "SmokeshellYellow" createVehicle [0,0,0];
             _smoke attachTo [_item, [0,0,0]];
         };
+        _item allowDamage true;
```

Comment:
What if the object was invulnerable before? Now parachuting makes it vulnerable.

Reply:
An `isDamageAllowed`-if-check would be the straightforward answer. Trouble is that this only works if vehicle is local :/ , which could be messy in MP.

Reply:
looks like you should use `statusEffect` functions like here https://github.com/acemod/ACE3/blob/9eba55be219f1b06d05f9050ca493062a460366d/addons/cargo/functions/fnc_loadItem.sqf#L44